web/src/components/wizard/config/ConfigurationStep.tsx

```diff
@@ -147,6 +148,18 @@ const ConfigurationStep: React.FC<ConfigurationStepProps> = ({ onNext }) => {
           />
         );
 
+      case 'textarea':
+        return (
+          <Textarea
+            id={item.name}
+            label={item.title}
+            value={item.value || ''}
+            placeholder={item.default}
```

Comment:
it was probably this way in bolt, but from what i read, it's an anti-pattern to use the default value as a placeholder. i don't think the KOTS config uses placeholders at all, so we can just not have placehodlers on text or textarea items.